test/integration/client-navigation/pages/head.js

```diff
@@ -16,6 +16,9 @@ export default () => (
 
       <meta content="my meta" />
 
+      {/* this will not render */}
+      <meta content={undefined} />
```

Comment:
It's not enough to introduce a tag here, a test has to be added too in `client-navigation/test/index.test.js`

Could probably be something like

```suggestion
      <meta property="empty-content" content={undefined} />
```

And then look up `meta[property=empty-content]`

Reply:
Yep, fully aware. I’ve spent 2+ hours trying to get the testing environment set up locally & it’s still not really working, that’s the hold up here.

Reply:
@ijjk could you add the test?